frontend/packages/topology/integration-tests/features/topology-toolbar-filter.feature

```diff
@@ -0,0 +1,17 @@
+@topology
+Feature: Topology Toolbar Filter Group
+              As a user, I should be able to use the Filter Groups on Topology page
+
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-tp-toolbar"
+              And user is at Add page
+
+        @regression
+        Scenario: Topology filter by resource: T-07-TC06, T-07-TC07
+            Given user has created workload "nodejs-ex-git-dc" with resource type "Deployment Config"
+             When user clicks on List view button
+              And user clicks the filter by resource on top
+             Then user will see Deployment and Deployment-Config options with 1 associated with it
+              And user clicks on Deployment checkbox to see only the deployment type workload
+              And user clicks on DeploymentConfig checkbox to see only the deploymentconfig type workload
```

Comment:
should be consistent i.e can make use of `DeploymentConfig` across

Reply:
That file is still in progress. We realized that it shouldn't be part of topology-list-view.feature and that's why I created this file. We are gonna update it later.

Reply:
all am is asking is to be consistent with your PR instead of using `Deployment-Config` in one place and using `DeploymentConfig` at other, use on i.e use `DeploymentConfig`

Reply:
Updated